packer/builder/azure/driver_powershell/builder.go

```diff
@@ -80,6 +83,8 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		"location":                  &b.config.Location,
 		"instance_size":             &b.config.InstanceSize,
 		"user_image_label":          &b.config.UserImageLabel,
+		"vnet":                      &b.config.VNet,
+		"subnet":                    &b.config.Subnet,
```

Comment:
I wouldn't spend too much time on it. I think the powershell version might be on its way out in favor of the REST API version. It seems to be very hard to make the powershell version do two packer builds in parallel, along with other challenges.